src/scripts/routes.js

```diff
@@ -229,57 +252,21 @@ define([
         autoFocus: false,
         transition: "fade"
     });
-    defineRoute({
-        path: "/mypreferencesmenu.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/menu"
-    });
-    defineRoute({
-        path: "/myprofile.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/profile"
-    });
-    defineRoute({
-        path: "/mypreferencesdisplay.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/display"
-    });
-    defineRoute({
-        path: "/mypreferenceshome.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/home"
-    });
-    defineRoute({
-        path: "/mypreferencesplayback.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/playback"
-    });
-    defineRoute({
-        path: "/mypreferencessubtitles.html",
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/subtitles"
-    });
     defineRoute({
         path: "/notificationsetting.html",
         autoFocus: false,
         roles: "admin",
-        controller: "notificationsetting"
+        controller: "dashboard/notifications/notification"
```

Comment:
wtf, why there are "notification" and "notifications"?..

Reply:
The latter is the list showing all of them while the former is the settings page for a single notification.